.github/workflows/ci_frontend.yml

```diff
@@ -32,12 +32,10 @@ jobs:
       - if: steps.changes.outputs.src == 'true'
         run: |
           cp config/environment.tsx.template config/environment.tsx
-          npm install
-          npm run lint
-          npm run build:p
-          npm run test
-          npm run pack:p
-        name: Build and Test
+          scripts/setup_ci.sh
+          scripts/ci.sh
+        name: CI script
```

Comment:
Should we change the name to something clearer? Such as `Build and Test` like the old step.